### ISSUE TITLE
List index out of range

### ISSUE THREAD

**Issue by nonstopfor** [facebookresearch/online_dialog_eval#4]
I just did as Readme.md told me. But I met a mistake. Here is the error message.

`Traceback (most recent call last):
  File "codes/inference.py", line 164, in <module>
    model_save_path = all_saved_models[0]
IndexError: list index out of range
`

**Comment by nonstopfor** [comment]
Also，there are some hard-coded paths which lead to some errors.

**Comment by koustuvsinha** [comment]
Hi, thanks for pointing this out. Have you unzipped the checkpoints in your folder such that `full_acl_runs` folder is in the root where you are running the inference?

**Comment by nonstopfor** [comment]
> Hi, thanks for pointing this out. Have you unzipped the checkpoints in your folder such that `full_acl_runs` folder is in the root where you are running the inference?

Yes, I have unzipped the checkpoints in my folder and full_acl_runs folder is in the root. And a strange thing I find is that, when I run "sh run_inference.sh", there is a problem of "List index out of range", but when I run "python xxx" directly, there is no such problem. I am confused by this.

**Comment by koustuvsinha** [comment]
Thanks, probably this issue creeped in due to my dev environment. I'll setup in a clean environment and debug this.

**Comment by koustuvsinha** [comment]
@nonstopfor did you solve this issue? Please let me know. Closing for now.

**Comment by nonstopfor** [comment]
> @nonstopfor did you solve this issue? Please let me know. Closing for now.

Yeah I solved it.

**Comment by koustuvsinha** [comment]
Awesome! Would you like to submit a PR with the change?

**Comment by nonstopfor** [comment]
> Awesome! Would you like to submit a PR with the change?

Emmm... It seems to be caused by the wrong path defined in "meta_tags.csv", so I just changed this file bud I didn't change your source code.

**Comment by koustuvsinha** [comment]
Ah nice good catch, yes `meta_tags.csv` is generated by Pytorch Lightning which dumps the configs that I ran in my experiments, hence the path is hardcoded. Thanks for letting me know! 👍
